Elastiflix/docker-compose-elastic-otel.yml

```diff
@@ -117,16 +117,21 @@ services:
     ports:
       - "5004:5000"
     environment:
-      - ELASTIC_APM_SECRET_TOKEN=${ELASTIC_APM_SECRET_TOKEN}
+      - OTEL_EXPORTER_OTLP_HEADERS=Authorization=Bearer ${ELASTIC_APM_SECRET_TOKEN}
```

Comment:
[for reviewer] the elastic token is not needed for otel as it's included in the auth header.